llvm/test/Transforms/Reassociate/fast-ReassociateVector.ll

```diff
@@ -370,10 +371,15 @@ define <2 x double> @test11_reassoc(<2 x double> %x, <2 x double> %y) {
 ; FIXME: shifts should be converted to mul to assist further reassociation.
 
 define <2 x i64> @test12(<2 x i64> %b, <2 x i64> %c) {
-; CHECK-LABEL: @test12(
-; CHECK-NEXT:    [[MUL:%.*]] = mul <2 x i64> [[C:%.*]], [[B:%.*]]
-; CHECK-NEXT:    [[SHL:%.*]] = shl <2 x i64> [[MUL]], splat (i64 5)
-; CHECK-NEXT:    ret <2 x i64> [[SHL]]
+; CHECK-CV-LABEL: @test12(
```

Comment:
nit: Perhaps remove the FIXME comment above now?

Reply:
The comment is still relevant when using the default flags (i.e. nobody is likely to see the "fixed" output yet) so I figured it's better to keep the comment until the new code paths become the default.